icons/pop_os.svg

```diff
@@ -0,0 +1 @@
656.300401.486184.333516zM16.166979 12.327926c-.0023 1.79057.173637 3.160113-.597312 3.323325-.625094.130092-.604258-.430498-.652876-.901201-.185212-1.82369-.354219-4.853709-.42599-6.6868603.05094-.7876622.567215-1.270194 1.104333-1.3884631.828828-.1821314 1.71322-.1300936 2.447126.3335156 1.451608.9177574 1.567365 2.6018928.687604 3.9761618-.481554.749817-1.937791 1.293849-2.562885 1.246542zm.185213-4.0140088c-.09029.014194-.238462.056769-.280135.1466535-.09955.2105163-.09261 1.5469413.08103 2.1075343.02778.09225.09029.179769.180583.212879.328753.122998.625093-.288574.759373-.536934.187529-.345344.247723-.752186.12965-1.130641-.111128-.352439-.456087-.8633552-.870501-.7994918zM7.8648071 9.203291c-.5440631 1.175582-1.4469761 2.109896-2.6971639 2.571141.3588503.957969.7246458 1.904111 1.0811807 2.859715.199104.515641.3889475 1.080967.2153102 1.589519-.1805826.520377-.884392.664659-1.3960428.111169C4.0841473 15.265696.75725919 8.4913197.51185202 7.9827686c-.2083646-.437591-.52091152-.9059291-.51165087-1.4121172C.0164073 5.8090068 1.1693581 5.0379016 1.7226819 4.6428873c.7732643-.5511277 1.6669168-.9272189 2.6230787-.9295843.7454824 0 1.382152.2081512 1.9470516.5771466.8658702.5676852 1.4678125 1.473616 1.6900686 2.4978159.1944736.9012012.1528005 1.823688-.1180737 2.4150255M5.3297046 7.9165389c-.1944735-.7214354-.6343545-1.3860988-1.1876781-1.852074-.1203886-.1017095-.2500377-.1963237-.4005233-.2483615-1.0371926-.3477072-.5880511 1.4239439-.4190443 1.8709954.2338315.6268184.5880511 1.4192182 1.0626596 1.8922852.1041822.104071.2268859.196323.3681108.224709.21531.04494.4144139-.108806.5232267-.288577.090291-.149013.1250188-.326418.1458551-.499088.039358-.36663.00463-.740354-.0926065-1.0998891zM20.26795 18.207522c-.0046 1.038396-.625093 2.076781-1.993354 2.076781s-2.004931-1.021831-2.004931-2.064954c0-1.300944.886707-2.105165 2.032713-2.105165 1.328903.01182 1.977149 1.019467 1.965572 2.093338zm-1.965572-1.438135c-.858925 0-1.340479.610263-1.345109 1.442865 0 .723799.395892 1.402656 1.328903 1.402656.93301 0 1.289545-.735626 1.289545-1.409751.01157-.700145-.351905-1.419213-1.273339-1.43577zm2.539733 2.982713c0 .153748.0046-.730894 0-.865719h.629724l.04399.541664c.20605.120634.525543.210516.787156.210516.486184 0 .993204-.125362.993204-.610261-.01621-.439957-.581105-.529834-1.037192-.577145-.77558-.08042-1.400674-.274383-1.400674-1.163754 0-.792395.877447-1.187412 1.530323-1.187412.525542-.01182.898283.07333 1.37289.461246 0 .319324-.0046.508548-.0046.827873h-.608886l-.03936-.525109c-.245407-.12063-.49776-.153745-.692233-.149016-.419045.0047-.865871.205789-.865871.565324 0 .352433.289395.449416.803361.525106C23.192008 17.91895 24 18.112908 24 19.025935c-.0046.99818-.921435 1.260728-1.70859 1.260728-.599627.0023-.974684-.111165-1.449292-.534563zM15.42 19.943693c0 .182132-.14354.328786-.321807.328786h-3.720465c-.178268 0-.321808-.146654-.321808-.328786v-.02838c0-.182127.14354-.328785.321808-.328785h3.720465c.178267 0 .321807.146658.321807.328785z"/></svg>
```

Comment:
This looks great! Is there any chance you could run this SVG through [SVGOMG](https://jakearchibald.github.io/svgomg/) or run `npm run svgo -- icons/pop_os.svg` on this, just to try and reduce the 'precision' (number of decimals) down to 3?
For example, here's my resulting path running it through SVGOMG (bear in mind I've not tested this!).
```svg
M21.446 13.104a.718.718 0 01-.301.456c-.255.163-.565.116-.785-.083-.206-.184-.276-.496-.21-.766.066-.281.28-.53.552-.608.61-.172.862.504.744 1zm-.373-1.92c-.378-.155-.248-2.177.097-4.632.097-.686.264-1.01.452-1.176.268-.234.757-.243 1.088-.194.38.057.743.237 1.018.509.243.241.271.487.188.806-.137.509-.37.996-.595 1.472a51.626 51.626 0 01-.581 1.128c-1.123 2.02-1.348 2.216-1.667 2.086zm-7.617-.912c.222.97.199 2.035.01 2.801-.179.724-.663 1.372-1.432 1.59-.673.191-1.458.052-1.94-.422-.791-.775-1.255-2.597-1.048-3.919.166-1.062.727-2.422 1.956-2.481.079-.005.157-.003.236.005 1.253.115 1.935 1.194 2.218 2.426zm-2.264 2.207c.07.012.139.012.208-.01.234-.068.368-.316.43-.557a1.968 1.968 0 00-.041-1.121 1.2 1.2 0 00-.296-.492c-.498-.466-.996.018-1.082.57-.067.43.03.93.295 1.277.122.16.298.3.486.333zm4.975-.151c-.002 1.79.174 3.16-.597 3.323-.625.13-.605-.43-.653-.9-.185-1.825-.354-4.855-.426-6.688.05-.787.567-1.27 1.104-1.388.829-.182 1.713-.13 2.447.333 1.452.918 1.568 2.602.688 3.976-.482.75-1.938 1.294-2.563 1.247zm.185-4.014c-.09.014-.238.057-.28.147-.1.21-.093 1.547.081 2.107.028.092.09.18.18.213.33.123.626-.289.76-.537.188-.345.248-.752.13-1.13-.111-.353-.456-.864-.87-.8zm-8.487.89c-.544 1.175-1.447 2.11-2.697 2.57.358.958.724 1.905 1.08 2.86.2.516.39 1.081.216 1.59-.18.52-.884.664-1.396.11C4.084 15.267.758 8.492.512 7.984.303 7.545-.01 7.077 0 6.57c.016-.762 1.17-1.533 1.723-1.928.773-.551 1.667-.927 2.623-.93.745 0 1.382.208 1.947.577a3.991 3.991 0 011.69 2.498c.194.901.153 1.824-.118 2.415M5.33 7.917a3.75 3.75 0 00-1.188-1.853 1.286 1.286 0 00-.4-.248c-1.038-.348-.589 1.424-.42 1.871.234.627.589 1.42 1.063 1.892a.755.755 0 00.368.225c.216.045.415-.109.523-.288.09-.15.125-.327.146-.5.04-.366.005-.74-.092-1.1zm14.938 10.29c-.005 1.039-.625 2.077-1.993 2.077s-2.005-1.022-2.005-2.065c0-1.3.886-2.105 2.032-2.105 1.33.012 1.978 1.02 1.966 2.094zm-1.966-1.438c-.859 0-1.34.61-1.345 1.443 0 .724.396 1.403 1.33 1.403.932 0 1.289-.736 1.289-1.41.011-.7-.352-1.42-1.274-1.436zm2.54 2.983c0 .154.005-.73 0-.866h.63l.044.542c.206.12.525.21.787.21.486 0 .993-.125.993-.61-.016-.44-.58-.53-1.037-.577-.776-.08-1.4-.274-1.4-1.164 0-.792.877-1.187 1.53-1.187.525-.012.898.073 1.373.461 0 .32-.005.509-.005.828h-.609l-.04-.525a1.506 1.506 0 00-.692-.149c-.419.005-.865.206-.865.565 0 .353.289.45.803.525.838.114 1.646.308 1.646 1.22-.005 1-.921 1.262-1.709 1.262-.6.002-.974-.112-1.449-.535zm-5.422.192a.325.325 0 01-.322.328h-3.72a.325.325 0 01-.322-.328v-.029c0-.182.143-.328.322-.328h3.72c.178 0 .322.146.322.328z
```